dtos/address.go

```diff
@@ -87,7 +87,7 @@ type MQTTPubAddress struct {
 	ConnectTimeout int    `json:"connectTimeout,omitempty"`
 }
 
-func NewMQTTAddress(host string, port int, publisher string, topic string) Address {
+func NewMQTTAddress(host string, port int, publisher string, topic string, authMode string, secretPath string, skipCertVerify bool) Address {
```

Comment:
don't modify the public API
add another function `NewMQTTAddressWithSecurity` for this

Reply:
updated.